README.md

```diff
@@ -323,6 +323,10 @@ as `onNext`, `onComplete`, or `cancel`.
 
 `take%%<n>%%<id>` : The driver should hang the test thread until it has received at least `n` items IN TOTAL, and then immediately cancel.
 
+`disconnect` : The driver should disconnect the client from the server.
+
+`resume` : The driver should resume the connection with the server.
```

Comment:
The case I was talking about was single client.  If a client get's too far ahead of the server then the resumption should be rejected.  So I'm curious what the plan is to allow testing that.

Instead of just testing successful cases.